### PR TITLE
Feature/add covers to tests

### PR DESCRIPTION
**Pull request by levelfivehub**
@Maks3w  

Following discussion, is this what we are after?  Do not merge, until rest are complete for this component.  This is just a sample - please add tag WIP.

What are your thoughts on group tag?

### CODE REVIEW

**Comment by Maks3w** [comment]
Only class level `@covers`. Method coverage may added in the future but not now.

File: test/SerializerTest.php [hunk 1]
```diff
@@ -15,24 +15,26 @@
 use Zend\ServiceManager\ServiceManager;
 
 /**
- * @group      Zend_Serializer
+ * @covers \Zend\Serializer\Serializer
```
Comment (Maks3w) [inline test/SerializerTest.php]: I think global prefix (leading backslash) is not required.

**Comment by Maks3w** [comment]
I prefer do only one thing by commit. Remove things is risky and require more time for review

**Comment by levelfivehub** [comment]
Re-review, is this what you mean?

**Comment by Maks3w** [comment]
it's reasonable. Anyway focus on add `@covers`. Rest of changes made can by done later in different PRs

**Comment by Maks3w** [comment]
https://github.com/zendframework/zend-inputfilter/commit/60434bc2ed16efacbdef57f269cf5b1d8817bfc2

**Comment by levelfivehub** [comment]
I see what you mean.  Watch out for next commit.

**Comment by levelfivehub** [comment]
I need to rebase and revert the other removal of group annotations.  Too many commits.  I will update you.

**Comment by levelfivehub** [comment]
Rebased.

**Pull request merged**